src/Microsoft.Health.SqlServer/Features/Client/RetrySqlCommandWrapper.cs

```diff
@@ -15,7 +15,7 @@ namespace Microsoft.Health.SqlServer.Features.Client
     /// <summary>
     /// A wrapper around <see cref="SqlCommand"/> to provide automatic retries for transient errors.
     /// </summary>
-    internal class RetrySqlCommandWrapper : SqlCommandWrapper
+    public class RetrySqlCommandWrapper : SqlCommandWrapper
```

Comment:
This has nothing to do with your code, but I noticed that the `ExecuteAsync` methods below capture the `cancellationToken` in their closures. As an optimization, we can avoid the (very slight) overhead by using the other `ExecuteAsync` overload that accepts a token directly.

Eg.
```csharp
_retryPolicy.ExecuteAsync(t => _sqlCommandWrapper.ExecuteNonQueryAsync(t), cancellationToken);
```

Furthermore, this will help avoid creating an unnecessary `Task` if the token is already cancelled by the time this method is called. Polly does something like:

```csharp
cancellationToken.ThrowIfCancellationRequested();
TResult result = await action(cancellationToken);
```

Reply:
Cool, thanks for pointing this out. This is tempting to fix now, but I think it belongs in its own PR where we could check all `ExecuteAsync` calls, so I created [AB#81043](https://microsofthealth.visualstudio.com/Health/_workitems/edit/81043).